webpack.config.js

```diff
@@ -200,5 +200,8 @@ module.exports = {
   },
   resolve: {
     symlinks: false,
+    alias: {
+      vue$: 'vue/dist/vue.esm.js', // needed because vue's default export is the runtime only
```

Comment:
We should be able to take this out once we switch to using SFC.

Reply:
What's SFC?

Reply:
[Single File Components](https://vuejs.org/v2/guide/single-file-components.html), so using `.vue` files. We can use the vue-template-compiler to compile them during the build process to only the runtime is needed.

Reply:
Ah, I see it's https://vuejs.org/v2/guide/single-file-components.html

Reply:
Indeed, that would be great because it saves a few bytes on Vue itself.